applications/callflow/src/module/cf_dynamic_cid.erl

```diff
@@ -43,9 +48,104 @@
                          )
          }).
 
+
 %%--------------------------------------------------------------------
 %% @public
 %% @doc
+%% Entry point for this module, based on the payload will either
+%% connect a caller to check_voicemail or compose_voicemail.
+%% @end
+%%--------------------------------------------------------------------
+-spec handle(wh_json:object(), whapps_call:call()) -> 'ok'.
+handle(Data, Call) ->
+    case wh_json:get_value(<<"action">>, Data) of
+        <<"manual">> ->
+            lager:info("user must manually enter on keypad the caller id for this call"),
```

Comment:
@stormqloud 
There are some minor alignment issues, if we resolve the other comments before you get to these we can fix the formatting once accepted.  However, notice this log line is indented by one space as is the lager line below.

There are also some alignment issues on line 93-94, 174-175, 181-184, 207, and in the block starting at 258.

Sorry if it seems picky but we try to hold all committers to the these same formatting standards :)

One other nit-picky note is we attempt to use all lowercase logging that is descriptive.  Typically we would not allow:
lager:debug("cid_id2 ~p ", [CIDKey]),
and request that the following line is lowercased:
lager:info("New caller id data : ~p",  [NewCallerId])